lib/failspell/version.rb

```diff
@@ -1,3 +1,3 @@
 module Failspell
-  VERSION = "0.1.1"
+  VERSION = '0.1.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.